tests/e2e/apps_test.go

```diff
@@ -303,7 +303,10 @@ var _ = Describe("Apps", func() {
 				Expect(result.State).To(Equal("STARTED"))
 			})
 
-			It("has the VCAP_SERVICES env var set", func() {
+			// TODO: Update the get app environment handler to return the vcap services secret ref. Negotiated as part
```

Comment:
All the discussion in the TODO should belong to an issue in the backlog. I believe they do, so the comment makes no sense. 
I also do not see the benefit of pending a test vs deleting it:
* If one wants to restore the test, git log is their friend
* pent tests are still compiled, so should be still maintained to at least compile in future refactoring
* while being pent, the test itself might become outdated and could trick anyone reading the code into not realising that the test is actually not run

Reply:
That makes sense. I'll remove it here and add a note to the relevant story that the test will have to be replaced once the functionality is in place.